include/boost/geometry/algorithms/detail/overlay/get_turns.hpp

```diff
@@ -143,7 +143,7 @@ class get_turns_in_sections
 
     template <typename Geometry, typename Section>
     static inline bool neighbouring(Section const& section,
-            int index1, int index2)
+            signed_index_type index1, signed_index_type index2)
```

Comment:
Yes, they will be renamed consistently in the whole library. For now in other places in this file the old name is used so I used it here as well to not mix them.